client/src/utils/fight/arrive.ts

```diff
@@ -48,7 +48,7 @@ const arrive = async (
 
   // Wait 0.25s before playing arrive SFX
   setTimeout(() => {
-    void sound.play('arrive', { speed: speed.current });
+    void sound.play('mybrutesounds', { sprite: 'arrive' });
   }, 250 / speed.current);
```

Comment:
I thought you didn't want to speed up sound depending on the fight speed anymore?